lib/Runtime/Debug/DiagObjectModel.cpp

```diff
@@ -3837,4 +3880,195 @@ namespace Js
         return FALSE;
     }
 #endif
+
+    //--------------------------
+    // RecyclableSimdObjectWalker
+
+    template <typename simdType, uint elementCount>
+    BOOL RecyclableSimdObjectWalker<simdType, elementCount>::Get(int i, ResolvedObject* pResolvedObject)
+    {
+        Assert(elementCount == 4 || elementCount == 8 || elementCount == 16); // SIMD types such as int32x4, int8x16, int16x8
+        Assert(i >= 0 && i <= elementCount);
+
+        simdType* simd = simdType::FromVar(instance);
+        SIMDValue value = simd->GetValue();
+
+        WCHAR* indexName = AnewArray(GetArenaFromContext(scriptContext), WCHAR, SIMD_INDEX_VALUE_MAX_NAME);
+        swprintf_s(indexName, SIMD_INDEX_VALUE_MAX_NAME, L"[%d]", i);
```

Comment:
If i can't be greater than 16 (2 digits) SIMD_INDEX_VALUE_MAX_NAME should be defined to 5.